core/src/main/java/org/infinispan/topology/CacheTopologyControlCommand.java

```diff
@@ -36,8 +36,8 @@
       JOIN,
       // A member is signaling that it wants to leave the cluster.
       LEAVE,
-      // A member is confirming that it finished the rebalance operation.
-      REBALANCE_CONFIRM,
+      // A member is confirming that it has finished the topology change during rebalance.
+      REBALANCE_PHASE_CONFIRM,
```

Comment:
Ok, I missed that you're sending a confirmation for each phase. Can you update the comment to mention this?